### PR TITLE
Hide actions for group rows

### PR DESCRIPTION
**Pull request by haynesjm42**

### CODE REVIEW

File: desktop/columns/Actions.js [hunk 1]
```diff
@@ -41,7 +43,9 @@ export const actionCol = {
     chooserName: 'Actions',
     chooserDescription: 'Record Actions',
     excludeFromExport: true,
-    elementRenderer: (value, {record, column}) => {
+    elementRenderer: (value, {record, column, agParams}) => {
```
Comment (lbwexler) [inline desktop/columns/Actions.js]: john -- I have a sinking feeling this is going to kill the action column for rows in our TreeGrid, which would *not* be acceptable....those rows definitely are bound to records

Comment (lbwexler) [reply desktop/columns/Actions.js]: can you please confirm this is not the case?

Comment (haynesjm42) [reply desktop/columns/Actions.js]: Rest assured that it is not the case 😄

**Comment by lbwexler** [comment]
OK.  Am going to merge it and sleep quietly then.  😴

**Pull request merged**